aten/src/ATen/native/DispatchStub.cpp

```diff
@@ -20,7 +20,7 @@ static CPUCapability compute_cpu_capability() {
     if (strcmp(envar, "default") == 0) {
       return CPUCapability::DEFAULT;
     }
-    AT_WARN("ignoring invalid value for ATEN_CPU_CAPABILITY: ", envar);
+    // AT_WARN("ignoring invalid value for ATEN_CPU_CAPABILITY: ", envar);
```

Comment:
what's the issue here? Why comment it out?

Reply:
It's mainly because LOG(ERROR) would be a better long term solution - it also emerges in fb infrastructure better as the standard logging mechanism. I removed Warning class for now but LOG(ERROR) is dependent on glogs move, so I will convert them in an immediate followup PR once this passes.